barcode_action/models/res_partner.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Creu Blanca
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+import json
+from odoo import api, models, _
+from odoo.tools.safe_eval import safe_eval
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.multi
```

Comment:
This should be `@api.model`.
(also in the readme)

Reply:
I do not agree, it should be multi, because we would like to use the res_id as part of our process. Usually it could be a @api.model (As in the example), but in some cases, it will be a @api.multi with a single record. For example, If we have a barcode for products, inside a SO we could add the "Add product". When you scan the product, it will be added on your SO, so the function will be assigned in `sale.order` and as a `res_id` you will send the active_id.